clientd3d/d3drender.c

```diff
@@ -19,6 +20,9 @@
 extern int main_viewport_width;
 extern int main_viewport_height;
 
+typedef std::chrono::time_point<std::chrono::steady_clock> steady_clock_time_point;
+static auto& chrono_time_now = std::chrono::steady_clock::now;

```

Comment:
This function renaming is a little odd, and only used once, so I'd suggest just using std::chrono::steady_clock::now in the one place it's called.